contracts/simple_airdrop/Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [package]
-name = "simple-astroport-airdrop"
+name = "phoenix-astroport-airdrop"
```

Comment:
I'm not sure that we have to rename crate's name

Reply:
simple airdrop didn't look like appropriate naming, lmk if i should change it

Reply:
Ok, makes sense